tests/components/light/test_mqtt_json.py

```diff
@@ -149,6 +149,46 @@ def test_controlling_state_via_topic(self): \
         self.assertEqual([125, 125, 125],
                          light_state.attributes.get('rgb_color'))
 
+    def test_controlling_color_with_templates_via_topic(self): \
+            # pylint: disable=invalid-name
+        """Test the controlling of the color with templates via topic."""
+        self.hass.config.components = ['mqtt']
+        assert setup_component(self.hass, light.DOMAIN, {

```

Comment:
As a minimum the test cases you worked on.

If you are up to it, all setup_components in this file would be appreciated. The tests are not 100% accurate without it.

If you prefer you can even make another PR for the others, just CC me in description and I'll review